cluster/local/integration_tests.sh

```diff
@@ -42,7 +42,7 @@ SAFEHOSTARCH="${SAFEHOSTARCH:-amd64}"
 BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-${SAFEHOSTARCH}"
 
 helm_tag="$(cat ${projectdir}/_output/version)"
-CROSSPLANE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}"
+CROSSPLANE_IMAGE="${PROJECT_NAME}/${PROJECT_NAME}:${helm_tag}"
```

Comment:
Does this really work?

Reply:
Yep! The commit that makes this change has more details, but you can see evidence of it working in the e2e tests that ran against this PR. It might make sense to have a more robust solution in the future, but this really isn't any worse than what we already had as we hardcode `crossplane/crossplane` in the helm chart.